CHANGELOG.md

```diff
@@ -14,6 +15,43 @@
 * Disable Cruise Control network resource goals when resource capacities are not set.
 * The `strimzi_resource_state` metric in the Cluster Operator is deprecated and is planned to be removed in Strimzi 0.51.
   Use kube-state-metrics based metrics from the [examples](https://github.com/strimzi/strimzi-kafka-operator/tree/main/examples/metrics/kube-state-metrics) as a replacement.
+* The User Operator does not ignore the ACL rules for the `*` and `ANONYMOUS` users by default anymore.
+  If you need to re-enable this feature, you can do it by configuring the `STRIMZI_IGNORED_USERS_PATTERN` environment variable to the following regular expression: `^\*|ANONYMOUS$`.
+  In case you deployed the User Operator through a `Kafka` custom resource, you can set it using the following YAML directly in the `Kafka` CR:
+  ```yaml
+  apiVersion: kafka.strimzi.io/v1beta2
+  kind: Kafka
+  metadata:
+    # ...
+  spec:
+    #...
+    entityOperator:
+      template:
+        userOperatorContainer:
+          env:
+            - name: STRIMZI_IGNORED_USERS_PATTERN
+              value: "^\\*|ANONYMOUS$"
+      #...
+  ```
+  In case you use the standalone User Operator, you can set the environment variable in its `Deployment`:
+  ```yaml
+  apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    # ...
+  spec:
+    # ...
+    template:
+      spec:
+        containers:
+        - name: strimzi-user-operator
+          # ...
+          env:
+            # ...
+            - name: STRIMZI_IGNORED_USERS_PATTERN
+              value: "^\\*|ANONYMOUS$"
+  ```
```

Comment:
I think it's too much having YAMLs/examples within the CHANGELOG. If we have them in the documentation I would just add a link there.

Reply:
You cannot really link to the docs here because the _release_ docs do not exist yet.

If you think it is too much, I can reduce the YAML, leave only the environment variable, and let the user to figure out how to set the environment variable.

But I think we have plenty of YAML like this already in the CHNAGELOG for previous releases. And I'm not sure why would it hurt to have the CHANGELOG verbose.

Reply:
Yeah I saw we have others, maybe they are not so verbose. Anyway no strong opinion, we could remove the second one about the standalone just mentioning they need to use the env in the Deployment but the value will be the same.